.eslintrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require('@amaurymartiny/eslintrc');
```

Comment:
Script to remove react plugins works, thanks!

Can you give an example of a `eslint-disable` that is a bug-catcher? I want to make sure I know which ones are the most important to address.

Reply:
> Can you give an example of a eslint-disable that is a bug-catcher?

Well, mainly the `no-unsafe-member-access` ones I guess. If it's `any`, then it might also be undefined, in which case member access will throw. Although we do `data && data.something` in most places, so should be fine.